src/main/java/com/amazon/opendistro/elasticsearch/performanceanalyzer/collectors/PerformanceAnalyzerMetricsCollector.java

```diff
@@ -65,6 +66,7 @@ public void run() {
           () -> StatExceptionCode.OTHER_COLLECTION_ERROR.toString());
       StatsCollector.instance().logException(StatExceptionCode.OTHER_COLLECTION_ERROR);
     } finally {
+      LOG.debug("{} took {} time to execute", collectorName, System.nanoTime() - startTime);
```

Comment:
can we change this to a metric ?

Reply:
+1, it would be good to have a "timetook" metric to indicate the time taken for metric collection.

Reply:
Already made the change in PA PR - https://github.com/opendistro-for-elasticsearch/performance-analyzer/pull/212